canonical/snapcraft.io
# feat: Add builds section to publisher pages

## Done
Moved the builds publisher section into the publishers React application

## How to QA
- You may need to register a new OAuth application (see [Hacking.md](https://github.com/canonical/snapcraft.io/blob/main/HACKING.md#github))
- Go to http://localhost:8004/<snap_name>/builds
- Check that you can connect a repo
- Check that you can disconnect a repo
- Check that you can trigger builds

## Testing
- [x] No testing required (explain why): Will do separately as this PR is already quite big

## Issue / Card
Fixes #
